procrastinate/app.py

```diff
@@ -1,6 +1,16 @@
 import functools
 import logging
-from typing import TYPE_CHECKING, Any, Callable, Dict, Iterable, List, Optional, Set
+from typing import (
+    TYPE_CHECKING,
+    Any,
+    Callable,
+    Dict,
+    Iterable,
+    List,
+    Optional,
+    Set,
+    Union,
+)
```

Comment:
Wow, I'm not used to having so many imports from a module they don't fit on a single line :D. Well pixel real estate is cheap and if Black wants to spend a bazillion lines, it may well.